src/System.ComponentModel.TypeConverter/src/System/ComponentModel/MemberDescriptor.cs

```diff
@@ -301,48 +321,60 @@ public abstract class MemberDescriptor {
         ///       NOTE: If you make a change here, you likely need to change GetHashCode() as well.
         ///    </para>
         /// </devdoc>
-        public override bool Equals(object obj) {
-            if (this == obj) {
+        public override bool Equals(object obj)
+        {
+            if (this == obj)
+            {
                 return true;
             }
-            if (obj == null) {
+            if (obj == null)
+            {
                 return false;
             }
 
-            if (obj.GetType() != GetType()) {
+            if (obj.GetType() != GetType())
+            {
                 return false;
             }
 
             MemberDescriptor mdObj = (MemberDescriptor)obj;
             FilterAttributesIfNeeded();
             mdObj.FilterAttributesIfNeeded();
 
-            if (mdObj.nameHash != nameHash) {
+            if (mdObj._nameHash != _nameHash)
+            {
                 return false;
             }
 
-            if ((mdObj.category == null) != (category == null) ||
-                (category != null && !mdObj.category.Equals(category))) {
+            if ((mdObj._category == null) != (_category == null) ||
+                (_category != null && !mdObj._category.Equals(_category)))
+            {
                 return false;
             }
 
-            if ((mdObj.description == null) != (description == null) ||
-                (description != null && !mdObj.category.Equals(description))) {
+            if ((mdObj._description == null) != (_description == null) ||
+                (_description != null && !mdObj._category.Equals(_description)))
```

Comment:
We used reference source as the source. We have the fix available, just haven't updated yet.